iocage_lib/pools.py

```diff
@@ -34,9 +36,15 @@ def health(self):
 
     def activate_pool(self):
         if self.health != 'ONLINE':
-            raise PoolNotActivated(
-                f'Please check pool status, it should be ONLINE'
-            )
+            # We should allow activating a pool if it's in DEGRADED state if it has active jails
+            # running - this can happen if a degraded pool was replicated but before replication
+            # there were jails running from it.
+            jails = json.loads(run(['jls', '--libxo', 'json']).stdout)['jail-information']['jail']
+            root_ds = self.root_dataset
+            if not any(jail['path'].startswith(f'{root_ds.path}/') for jail in jails):
+                raise PoolNotActivated(
+                    'Please check pool status, it should be ONLINE'
+                )
```

Comment:
Whats the problem in activating on any DEGRADED pool?

Reply:
Old iocage behavior, should we change it perhaps ?

Reply:
I think so, perhaps simply add DEGRADED to an allowed status